src/main/java/com/minecolonies/commands/CommandEntryPoint.java

```diff
@@ -0,0 +1,56 @@
+package com.minecolonies.commands;
+
+import net.minecraft.command.CommandBase;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Command entry point to make minecraft inheritance happy.
+ */
+public class CommandEntryPoint extends CommandBase
+{
+
+    @NotNull
+    private MinecoloniesCommand root;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private field 'root' could be made final; it is only initialized in the declaration or constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3AImmutableField)